src/components/plots/stackedPlot.ts

```diff
@@ -62,10 +62,16 @@ module Plottable {
     public _updateStackExtents() {
       var valueAccessor = this._valueAccessor();
       var keyAccessor = this._keyAccessor();
+      var filter = this._isVertical ? this._yFilter() : this._xFilter();
+
       var maxStackExtent = Utils.Methods.max<string, number>(this._datasetKeysInOrder, (k: string) => {
         var dataset = this._key2PlotDatasetKey.get(k).dataset;
         var plotMetadata = <Plots.StackedPlotMetadata>this._key2PlotDatasetKey.get(k).plotMetadata;
-        return Utils.Methods.max<any, number>(dataset.data(), (datum: any, i: number) => {
+        var data = dataset.data();
+        if (filter != null) {
+          data = data.filter((d, i) => filter(d, i, dataset, plotMetadata));
+        }
```

Comment:
We could, but I'm not sure that's cleaner. I find in general that ternaries that don't evaluate to simply a constant value are kind of hard to read. The way I wrote the code reflects this logic:
 - get the data from the `Dataset`
 - If there is a filter, use it to filter the data.